lib/inv_bicgstabl_quda.cpp

```diff
@@ -11,144 +11,287 @@
 #include <dslash_quda.h>
```

Comment:
This file could also do with some cleanup to remove the dead code, e.g. lines 166-173, 
225-227, 271-276, 686-689, 711-714.

Reply:
https://github.com/lattice/quda/commit/7dfe43652b77b3310e67d99b4b85d51d4b8194bf